### PR TITLE
Moving the claim-formats section out to a distinct, PR-able registry

### LINKED ISSUE #340: Registry for VP and VC types
The PE spec currently "hard-codes" format identifiers for a couple of credential and presentation formats. This prevents implementers from using PE with other formats. As discussed at IIW, I suggest to introduce a registry to allow for extensibility. 

Here are examples of how PEW could be used for other credentials: https://openid.net/specs/openid-connect-4-verifiable-presentations-1_0.html#name-alternative-credential-form

### PR DESCRIPTION
**Pull request by bumblefudge**
This would close #340.

### CODE REVIEW

**Review by csuwildcat — APPROVED** [review]

**Comment by brentzundel** [comment]
I won't have time to review this until next week, please hold off merging it until then.

**Review by brentzundel — APPROVED** [review]
very minor nit

**Review by bumblefudge — COMMENTED** [review]
LGTM

**Pull request merged**
